tests/models/decoder_only/language/test_cls_models.py

```diff
@@ -0,0 +1,53 @@
+"""Compare the outputs of HF and vLLM when using greedy sampling.
```

Comment:
Can we move this to a new directory (similar to the one for embedding models), and update `test_pipeline.yaml` so that this is tested under Other Models Tests?

Reply:
Just moved under embeddings